ocp_resources/project_request.py

```diff
@@ -2,6 +2,7 @@
 
 from typing import Any, Optional
 from ocp_resources.resource import Resource
+from ocp_resources.project_project_openshift_io import Project
```

Comment:
what about Project in project_config_openshift_io?

Reply:
Should we use it instead? Before the class-generator PR, cleanup called the `Project` from `project_project_openshift_io`

But I do see that even with this PR, unprivileged user can't cleanup the project...

Reply:
I think this is only relevant to project.openshift.io, based on what I see 
```
[cloud-user@ocp-psi-executor-xl ~]$ oc explain ProjectRequest
GROUP:      project.openshift.io
KIND:       ProjectRequest
VERSION:    v1
```